readthedocs/config/tests/test_config.py

```diff
@@ -737,6 +737,26 @@ def test_requirements_file_respects_configuration(tmpdir):
     assert build.requirements_file == 'requirements.txt'
 
 
+def test_requirements_file_is_null(tmpdir):
+    build = get_build_config(
+        {'requirements_file': None},
+        get_env_config(),
+        source_file=str(tmpdir.join('readthedocs.yml')),
+    )
+    build.validate()
+    assert build.requirements_file is None
```

Comment:
Ups, forgot to rename this to the latest changes, I'll fix this in a couple of minutes